.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     'jest': true,
   },
   'rules': {
-    'no-use-before-define': 'on',
+    'no-use-before-define': 1,
```

Comment:
please set this to `"error"`. Currently it is set to `1` which means only warning. Setting it to error will ensure a better code quality and will be more semantic than writing `2`.